### ISSUE TITLE
Patch Notes 1.06

### ISSUE THREAD

**Issue by LeflairKunstler** [BeatAroundTheBuscher/ROSIGMA#57]
1.06 (November 2021):

Genestealer Cult (GSC) notes:
Implementation Split into Phases, simplest stuff first.

Test list:
GSC Civilians
GSC Spaces / Space Hulk mission 7 / artifact site.
GSC Arbites lists
4-tile unit corpses 

Autoinjector/Genestealer kiss attack from GSC medics and genestealers:
How it works vs PDF/Sisters/Guard/Marines.

Added some generic small arms.
- Pump-Action Shotgun (better range/accuracy, but higher TU cost (lower rate of fire).
- Generic Heavy Stubber (slightly less rapid-firing)
- AutoStubber (SMG) with long ammo magazine. (6 rounds/autoshot, 3 rounds/snap shot but low accuracy/range).
These generic small arms are linked to a research topic, 
    requiresBuy:
      - STR_STANDARD_SMALL_ARMS_RESEARCH
Which right now has no unlock, but we can enable it if we want. Purpose: To de-clutter the buy screen.

Also added:
-Webber pistol
-Heavy Webber (with ammo). AoE attack. Might look funky

Stun Shells and Stun Gas Shells for shotguns (combat) and the pump-action.

Done:
-Genestealer Kiss. Genestealers get the ability to zombify units. Units affected will turn traitor and join the genestealer cult.
-Infector Autoinjector kit for Infected Medics etc. Similar but less capable than the Genestealer Kiss. Tainted fluids inject the gooey xeno seed into the victims body, making them see things in an entirely different light. Arbites and Guard have unique zombie units and sprite assets.

Added mixed genestealer with infected voidcrew lists into the existing space hulk special missions.

To Do Research:
Research Chain 1A-5A (Cult presence, origins and purpose + spawn a base / mission chain to get rid of them? Work in Buschers World domination/spread script?
Research Chain 1B-5B (Infection/genestealer cult lifecycle fluff.)

To Do Equipment:
GSC arbites stun rounds for shotgun.
Heavy webber.
Webber pistol.
Gas grenade / stun grenade.

-Needle Guns and Goads. 
-Low level lists: The Infected, reusing existing art assets but creating new units and armor code + inventory art at least, we can introduce various infected GSC members to fight. Suitable for early foes like turned Civilians, Arbites, PDF, guard etc. Stub guns, autoguns, shotguns, webbers, some grenades, suicide troops, melee etc. 
I´ve done up about 30ish inventory images, so anyone that feels like pitching in can write up some Infected/Brood Brother type units and armor.
Does need some ufopedia art, but we can make some suitable placeholder to keep the workload sane.

GSC Units:
Finished: 4 basic civilians
2 Penitents
2 PDF
1 PDF Medic Infector.

Also done:
  - type: STR_GSC_SPACER_FEMALE #Infected Crew, space hulks etc 
  - type: STR_GSC_SPACER_MALE #Infected Crew, space hulks etc
  
  - type: STR_GSC_OGRYN #Infected 
  - type: STR_GSC_OGRYN_ORANGE #Stronger version. 
FLAK female
FLAK male
  - type: STR_GSC_CARAPACE_F #Infected STormtrooper female 
  - type: STR_GSC_CARAPACE_M #Infected STormtrooper male 

SCION m/F
  
  - type: STR_GSC_COMMISSAR F/M
  - type: STR_GSC_COMMISSAR_CARAPACE  F/M
  
  - type: STR_GSC_OFFICER #Infected Guard Officer
M/F
Carapace versions.

Psyker M/F

Wheeled HB and autocannons.
  
  - type: STR_GSC_SQUAT_FLAK #Infected Squat 
  - type: STR_GSC_SQUAT_CARAPACE  #Infected Squat Carapace
  - type: STR_GSC_JUMP_MEDIC #Flying infector
  - type: STR_GSC_WITCH #Infected Psyker, in lore needed to produce a Magus.  
  
  - type: STR_GSC_ARMORED_CAR #Heavy Stubber, twin heavy stubber? With AP rounds at last tech tier? Use the bulletdesigner armored car sprite. HB version done with existing weapon (Guard mounted). Needs fixing up the battlesprite.
  - type: STR_GSC_SENTINEL #Heavy bolter variant, reusing the Arbites version.

Not done:
  - type: STR_GSC_BROOD_BROTHER
  - type: STR_GSC_BROOD_SISTER
  - type: STR_GSC_JACKAL_ROUGH_RIDER #female Single tile bike? Not sure if possible 
  - type: STR_GSC_QUAD_BIKE #lobs grenades? Molotovs? Gas? There´s a quad bike in other mods, modification of that? 
  - type: STR_GSC_CULTIST_MOB #Four tile unit using the duplication of sprites technique? Moving as one, linked minds. 

  #GSC Infested Adeptas Abbey forces. Basic ones are useful, rest are not high prio. Most can be handled with KISS mechancis (turn traitor)  
  - type: STR_GSC_REPENTIA #Infected version done
  - type: STR_GSC_FRATERIS
  - type: STR_GSC_NOVICE
  - type: STR_GSC_ADEPTAS #Infected version done
  - type: STR_GSC_PILOT
  - type: STR_GSC_ELOHIM  
  - type: STR_GSC_RETRIBUTOR
  - type: STR_GSC_MEDICAE  
  - type: STR_GSC_HOSPITALLER
  - type: STR_GSC_SERAPH
  - type: STR_GSC_CELESTIAL
  - type: STR_GSC_DOMINION
  - type: STR_GSC_INQUISITOR
  - type: STR_GSC_CANONESS
  
  - type: STR_GSC_NEOPHYTE_HYBRID #Mainline GSC infantry. Tougher than Brood Brothers. 
  - type: STR_GSC_ACOLYTE_HYBRID #big bulb head proper
  - type: STR_GSC_ACOLYTE_HYBRID_HEAVY #Power saws and Mining Laser, heavy weapons
  - type: STR_GSC_CLAMAVUS #VOX Propaganda buffer/debuffer com-hacker female? Get out of here G.I.
  - type: STR_GSC_BIOPHAGUS #A master of gene-manipulation and bio-alchemy, Injector Goad, Needle Gun Pistol infector, grenades etc. Doctor, female the Injector Goad to hurl vials of corrosive acids and flesh-morphing mutagens that melt their foes into a protean ooze. 
    #Can use the Psyker coat sprite for this one, turn it white. 

  - type: STR_GSC_FAMILIAR #lil multiarmed bugger
  - type: STR_GSC_ABERRANT_SMALL #Four Tile, Ogryn sprite modification 
  - type: STR_GSC_ABOMINANT #Four-tile unit aberrant
  - type: STR_GSC_HYBRID_METAMORPH #Melee claw focused
  - type: STR_GSC_HYBRID_INFECTOR #Bulbhead, feeder tentacle mouth ala cthulhu spritesheet. Zombifies rather than turn units traitor. 
  
  - type: STR_GSC_SABOTEUR #Female
  - type: STR_GSC_KELLERMORPH #Gunslinger
  - type: STR_GSC_SANCTOS #Assassin, cloaked. Blade and darts. Stealthy. Needler acid sniper rifle. Deployable Familiar. 
  - type: STR_GSC_NEXOS #Officer, buffs TU to nearby units ?
  
  - type: STR_GSC_MAGUS	#Female Psyker Leader/Commander. Mind Control, AoE Mass Hypnosis Stun attack. Psionic Blast, more direct attack. 
  - type: STR_GSC_PRIMUS #Male Commander, life drain melee bone blade   4
  
  - type: STR_GSC_YOUNG_GENESTEALER #Purestrain, lighter hues (pink instead of purple, bone white instead of dark armor etc). 100-120 HP. Fast, high reactions, lethal in melee. No Kiss. 
  - type: STR_GSC_ADULT_GENESTEALER #Purestrain, already exists. MAybe make it more purple. Chonky health, lethal. 200ish HP. Genestealer Kiss to turn player and civvie units. 
  - type: STR_GSC_YMGARL_GENESTEALER #Feeder tentacle head. Active Camo, very stealthy. 
  - type: STR_GSC_VECTORI_GENESTEALER #Stealth, very high reactions, very fast. Not very durable (less armor). 
  
  - type: STR_GSC_BROODLORD  
  - type: STR_GSC_PATRIARCH #Four-tile genestealer psyker commander 1000HP

Suitable raceLists:
-Infected civilians, stub pistols, melee, backed up by some "heavies" with shotguns/autoguns/grenades and suicide belts. Commander tier could be a genestealer or some sort of Hybrid, but also an infected Priest or Officer. Ogryn is also a possibility. 
-Infected Arbites, riot unit (Shields)
-Infected PDF (autoguns and heavy stubbers)
-Infected Guard garrison with infected Ogryn for spice. (lasguns, some heavier weapons. Maybe a wheeled heavy stubber or cannon, or captured Sentinel).
-Infected Sisters/Repentia. Not a priority as I´m thinking the GSC will be mainly to flesh out the early game.
-Added Missions. While these lists can show up in existing mission rotations like cultist activity or Riots, we can also add in some new ones. 
In the lore the GSC captures civvies to infect, The Chaos Cult ones would need some replacing, "sacrifices" can remain but instead be infected by a genestealer (spawn a genestealer at a grate that then goes and infects the nearby civies stuck in the room). 
Cathedral map could also be reused for the GSC as either cult gatherings/base or some sort of attack on a local Imperial church gathering.
Sabotage, using some more elite infected/Brood brothers and Hybrids/Saboteurs, the player needs to protect some sort of Imperial installation/Ammo depot. 
-Guard Base/Outposts can be resued, hosting turned Arbites/PDF/Guard lists. Either the cult is attacking the base, or have already taken it over and needs to be purged.
-New maps underground once Bullet Designer is done with the new set. Suitable for the cult base or just "purge" missions.
-Scripting: To spice this all up, we can add some GSC base that spawns these missions, and starts building more bases, if left ignored they spread quickly and adds lots of negative score per month. 
The Base can upgrade with levels as well, going from civies and other infected, to eventually the Brood Brothers/Hybrid mix, and genestealers (toss in some turned guard elites or Sisters).

1.06
Fixes:
• Tzeentch Witch Flying Mode now comes with a bottom part.
• Some Tzeentch armors (And one Nurgle) anti camo was not being applied correctly.
• Improved General Stores and Laboratory descriptions (the work of Ettykitty)
• Fixed damage range of Harborym and Throne rockets
• Fixed Hellspite and mephisto+ammo buy requirements for tech.
• Made Galgalim ammo have the same requirments as the weapon
• Fixed light melta ufopedia crash
• Changed Potestas potassium melta gun snap name to be reflective of the correct type of melta
• Normalized use and unlock Potestas requirements
• Changed Potestas ammo to use the intended boxammo
• Added Codex Entry for Light Chainsword and a Missing String.
• Added needItem for STR_TZEENTCHS_SISTERCHAMP (Tzeentch Sisters Commander)
• Fixed Blue Horror Kill Tokens reward to require Blue Horror Corpses instead of Flamers
• Fix Female Krieg riding Mounted Weapons
• Necron Live Capture Disabled
• Fix Duplicates in alienRank for Traitor Guard Bases
• Adeptas twincore plasma can now be manufactured instead of the foul chaos version. The responsible secret cultists have been purged.
• Standardisation of certain Adeptas weapons requirements, ufopedia articles and manufacturability.
• Adeptas no longer tempted to put on the Tzeentch sorceress outfit and lead the game to ruin.
• Celestian armor language lines updated to read "Celestian" instead of Celestin.
• Deathcult assasin armor energy recovery increased to 60 as intended.
• Dominion armor manufacturing time reduced by a factor of 10 (a 0 too much)
• Second Power Shot for the Lucius Heavy Lasgun renamed to POWER SHOT!
• The arcing Flamers are now no longer capable of ctrl-forced heavy artillery support (through power drop off).
• All versions of the Thunderhawk will now use Prometheum to refuel.
• Yarrick somewhat more manly, unrelated: he´s switched hands for his power claw.
• IG training mission only available to Guardsmen now.
• Holograms given some language strings.
• Terminator ransom normalized
• TFTD patch fix applied to GRUNGE map (used for Space Hulk lvl 2) with a hole in a second level floor.
• GRUNGE map elevator are now shoot through
• Made statCaps consistent between Guard soldiers.
• Halved Base Mission Weights for all old factions (makes sense with other campaign changes and new factions, to reduce spam). 
• First alien mission similar to vanilla xcom reenabled
• First month terror mission reactivated with a easier terror mission than the standard ones
• No Chaos Air Superiority missions in first month and only two crafts per month
• No TG/Cultist bases in the first month

Special thanks to Filip H for a lot of testing, streaming and minor corrections over the last months. Thanks Tarqeq for pointing out some inconsistent stat caps, Qweals, Kildarien and the rest of you lot for all the playtesting, streaming etc. Makes it a lot easier to find things to fix and tweak. 

Changes:
• Broke out Tzeentch lists for mission generation and lowered CRAFT/UFO spam in general. Mission wave chains will now be shorter to allow for several different raceLists to be used per month instead of just one. We´re still lacking some early game lists, but we´ve made the first few months a lot easier vs 1.05. We´re looking into ways to do customized enemy mission lists for a harder difficulty (especially if playing Space Marines).
• Enemy Score values updated and added to units that were lacking them. Enemy heavies, specialists etc worth more. Chaff worth less. Score values: 1-10 for early game chaff units, leaders topping out at 15-20. Midgame units at 15-25, CSM at 25-30, values top out at 40-50 for the high end CSM Lords, Dreadnoughts etc.
• Enemy Base Building recoded to work a little different. You won´t notice.
• Chaos Turrets reactions and accuracy reduced from 120 to 60/80, making them easier to approach for hugs.
• Turret Armor buffed 'bout 30 points for Chimera, Taurox, Tauros, Leman Rus, Exorcist and Immolators.
• Beastmen flak armor front and side values buffed by 10 points.
• Scions token cost is now 90 instead of 120.
• Deserter mission type now expanded to escalate over months, you will no longer be buried in Squats on month 0 (in fact, Traitor Squats have been pretty much removed from Deserter lists).
• Added non-Hotshot Tech Tier for some enemies.
• Less aggravating Chaos Hotshot Power
• Made some early tier enemy lists easier (more normal traitor guard, fewer squats)
• Low level enemy squats have a 5 point armor nerf to the front and flanks. Rear armor weakened by 10. Resistances changed to make them 20% weaker to melta weapons and soldier tier squats are 10-20% weaker to AP (normal las/bullets included) rounds.
• Enemy Squats have 5-10 points lower reaction score.
• Chapel (Psi labs) requirements are now LEADER_PLUS (interrogatation of a leader), MIDTIER, and STR_EXAMINATION_ROOM (Chaos Chapel), making it possible to get this earlier.
• Commendations depend on Wins now instead of just participating
• Updated 2 racelists to reduce dreadnought spam during terror missions
• Updated Chaos Marine lists and removed some place holder units. Replaced with Black Legion CSM.
• Reduced frequence of Alpha Legion, removed from some lists.
• Reworked basic CSM to have handheld lascannons instead of heavy plasma (moved to Black Legion CSM instead).
• Replaced chaos longlas with tz version for combat servitors
• Decreased Imperial Cache Value from 200 -> 100, added it to terror missions in the ammo depot.
• Obliterators weapons buffed, power +20, they now do armor rend damage before pen calculation instead of after, and have radius 4 instead of 3.
• Enemy Chaos bases now have more defenders.
• Traitor Guard base maps are now slightly bigger to reduce overcrowding.
• Enemy retaliation missions now have more attackers.
• Enemy retaliation missions now have more attackers (as does battleships).
• Final Mission now have more defenders in all stages. Bring plenty of ammo.
• Renamed Tzeentch lasguns to Nihilus Pattern (can be used by some other enemies).
• Fire now gets doused before Fatal Wounds are healed.
• Arbites Sabotage missions now have enemy lists divided by months, making for a easier experience in the first 6-8 months.

Bolter ammo rebalances:
• Basic light bolter ammo nerfed slightly, 2% less to armor damage (pre armor calc). -5% AP and -5% to Health. Still slightly better pen than standard bolter ammo (but standard bolter ammo has 5 more power so).
• Mastercrafted light bolter ammo nerfed slightly, -5 power (to 75), -0.10 AP, ToArmor damage -0.1
• Mastercrafted standard bolter ammo buffed slightly, +0.05 ToArmorPre, +0.05 AP. Power is 85.

Adeptas:
• Enemy Seraphim armor types added to units that can unlock flying armor tech for the player.
• The Sister Superior can now use more armor types.
• Adeptas Malthus light lascannon, Harmonic melta, Dominion bolter, Galgalim Assault cannon, Dominion Power axe costs have been changed. Several of the high tier versions can now only be manufactured instead of also purchased. For the remaining weapons that can be both purchased and made, the purchase option is much more cash expensive but the manufacturing option requires materials.
• The Adeptas Power Axe is now high tier instead of midtier. The Halberd is now the midtier dedicated melee option.
• Sister Superior promotion minrank changed to 3 from 5, allowing the fielding of more Sister Superiors.
• Sister Superior can now use armor sets that the Adeptas/Canoness can use (most would be a downgrade, but hey).
• Novice promotion now requires rank 2 (Cantus) instead of 3 (Novitiate). Purely due to how the openxcom ranking system works and for gameplay ease, so don't mind if its a slightly unfluffy.
• Adeptas Dominion armor now requires high tier, mastercrafting and the honorguard research.
• Celestian and Dominion armor shields dropped to 40 from 83 HP. Plasma is now 10% more effective than it used to be vs these armors (They are still very good armors).
• Light Handflamer toHealth damage increased to 1.3
• Harmonic melta ammo now requires promethium to produce.
• Harmonic melta ammo has a new bigOB inventory sprite.
• Energy recovery increased for some lazy armors.

Maps:
• Added hive city and rig city maps to new STR_CIVILIAN_TERROR mission, a script randomized if this or the ordinary military-targeting terror mission happens (80% chance per month for one of them to be picked). STR_CIVILIAN_TERROR has civilians running around, so less armed help!
• Added Height: 8 and 9 to some Maps.
• Elevators in Space Hulk Purge can now be shot through
• Turned Fences from N/W Wall to Object (this means that they can be destroyed by melee now and stop movement properly).
• Updated URBAN06-08 and 16 with more cover and spawn points.
• Modified several urban maps, the Cathedral maps, the final mission map and the Space Hulk top floor map with more spawn points, routes and cover, alternative entry points.
• Fixed a final mission pyramid where units could get stuck if the map tile spawned at the right edge of a map.
• Replaced the 6 power generator objects from the Space Hulk top floor that would provide a [lot] of money.
• Added a pathable Hive Map.
• Added over 30 new Chaos UFO/Craft. Both to existing classes of craft, as well as a few new ones. New maps for everything but the smallest and largest classes of ships. UFO spawning redone to fit these new craft, some have replaced duplicate existing slots (i.e, you will see less of the base mod ships, more variety) while others are new. Expect some more UFOs to be flying around.
• 9 new Medium Scouts (dual small craft in formation + a deployable watch tower version)
• 8 new Large Scout (one heavy large scout and a deployable connected twin watch towers version).
• 2 new Supply Ships, one fast and one large version.
• 9 new Strike/Attack cruisers, similar in size to the heavy large scout.
• 2 new Terror ships. One Fast and one Heavy.
• 1 new Heavy Harvester.
• 1 Mobile Modular Fort.
• Added 10 new Imperial/Arbites UFO craft. 5 Small/Large Scouts, 2 medium transports (2-3 floors), 2 large transports/terror ships and 1 battleship (5 floors). These are primarily used by the new enemy Genestealer Cult faction (See below).
• 1 New mobile industrial rig craft. Rather large pyramid shaped thing.
• Added 2 new Taurox UFO craft. Dual Taurox, and a quad Taurox Convoy with a extended big Taurox limo variant.
• Added 2 new Truck UFO craft. Dual GSC Trucks (one armored) and a quad convoy with 1 Taurox escort.

Guard:
• Multilaser damage buffed to 60 from 50 per shot, given same standard damage type as other non lascannon las weapons.
• Now has access to Light Bolt Pistol De'az.
• Veteran Officers get a sweet beret now (thanks to our friends at IDT Filip H and EfreneSpartano)
• Lasguns have more ammo
• Priest's ability was turned into Fanaticism. They keep their melee buffing. Fanaticism allows affected soldiers to withstand the naughtiness part of Daemonettes as well as Chaos Spells for one turn
• IG brings another Chimera instead of a Hydra to the battlefield
• Commissars can execute anyone and not only traitors. The morale will be restored next turn. The more worth a soldier is, the more the action instills fear into the hearts of the Guardsmen
• Scions only cost 90 instead of 120 Imperial Tokens now

Arbites:
• Added soldierTransformation Pardon Penitent for Arbites
• Require Lascannon for Arbites Sentinel/Lascannon
• Changes to Arbites armors: Medic Armor uses normal Enforcer, Shield uses Heavy Armor storeItem
• Arbites Armor got a buff.
• Fixed Requirements for Judge Bike and Bike Ammo ( STR_HP_CANNON_SHELL).
•  Given same built in melee bayonet ability as Guard. 
•  Added Missing Recoil Tag for Arbites Heavy Stubber 
•  Arbites Penitents stats now matches the IG Penitents. 
•  Sentinel/Missile has -40 Weight now to accomodate two Missile Packs (same for IG) 
•  Arbites Vehicle Armor (turrets) buffed to match new IG armor values. 
•  Arbites Chimera Heavy Bolter now has more Dakka and less sniping, like the multilaser in role. 
•  Arbites now have access to the Light Bolt Pistol De'az pattern. 
•  Restructured Arbites startingBase.

New:
• New events around mysterious activity and earthquakes. You might be able to figure out what's going on there.
• CAS Valkyrie added, this craft floats above the battlefield, allowing for tacticool operator insertions and rocket barrages.
• Added more Deserter Guards. Grenadier, Gunner, Officer and Psyker. Intended to serve as weaker enemies (although the Psyker can be quite dangerous). They use mainly standard guard equipment, autoguns, las-weapons, grenade launchers, heavy stubbers, chainswords etc.
• Added Khorngor (+different inventory art) to mid and late game Khorne lists, beefier cousins of the Ungor, they have more health and are armed with shotguns, ripper guns and chainaxes.
• Added a Bloodgor Warband list for the midgame, will eventually have more flavors of Khorngors, Bloodgors, Minotaurs etc.
• Tzeentch Traitor Guard added. Basic trooper, gunner, scout-sniper, commando, carapace and officer. Their terror units are the various Tzeentch servitors (combat, battle-plasma). Can be expanded in the future with weapons platforms, plasma sentinel, their own Chimeras and what not.
They use the Nihilus pattern lasguns added in 1.05B and the new longlas.
Tzeentch lists broken out for their own mission cycle, allowing for shorter waves of UFO/Craf and more variety in enemy lists encountered.
• Added Tzeentch Traitor Guard racelists (quite a few) to rotations. They favor research type missions, and are a step above normal traitor guard in terms of accuracy (But not health or reactions) and techy equipment.
• Added Pink Horrors to Tzeentch. These beefy Horror Daemons spawn two Blue Horrors on death (Pink - Blue - Yellow). Pink Horrors are capable spellcasters (currently using similar spells to Blue Horrors). Certain Tzeentch enemies can also summon these lesser daemons.
• Added Tzeentch Long Las (Nihilius pattern) and Twincore Plasma. Long ranged, accurate and powerful weapons for enemy sharpshooters. Rubric Marines have been given the Twincore Plasma at last tech level.
• Added Black Legion CSM "soldier" with its own art assets. This late game elite CSM is tougher than his "basic" CSM counterparts and very well equipped. Shows up with a Chaos bolter with MC-tier ammo, twincore plasma rifle or the heavy plasma. Carries powerful grenades. Has some resistances (20%) to basic weapons, but melta and melee will still work fine. Shows up in late game lists (and the final mission lists!) but can also appear with certain late Blackstone configurations. Basic CSM has been de-equipped of high tier plasma weapons to compensate.
• Final Mission have gotten two unique enemy lists, one for the power armor player factions and one for Guard/Arbites. These are composed of a "Chaos United" crop of enemies, facing you off against a wide variety of late game nasty chaos folks.
• Added 3 new civilians. Tan, red and orange. They have GSC zombie versions (see below)
• Added new Genestealer Cult infected lists. Voidcrew (hello Ripley), Civilians, Arbites (nearly full set), PDF, Guard and Adeptas.
• Genestealer attacks are now augmented with Genestealer Kiss, capable of infecting (And zombifying) most units! Added dozens of new infected versions of existing player units + new inventory art for them (and sometimes unique battlesprites).
Zombified units will pick up their guns and use them against you!
• GSC Medicae type units has a autoinjector weapon with genestealer tainted fluid that can infect units, take them out!
• Augmented existing genestealer space hulk mission with several lists of mixed infected voidcrew. These enemies use mostly lasguns.
• Added a dozen other GSC related lists, arbites, pdf etc.
• GSC will establish bases, primarily in Arbites Precincts and target the surrounding population. At a low level, their bases will not launch many missions but as they grow they´ll starting infiltrating other Arbites Precincts, launch kettling round ups of civilians in urban centers for infection, capture and use disguised transports to move around the geoscape for infiltration missions etc. They will also target PDF units and deserts for infection, and if agitated enough by the player will plot retaliation missions + launch base attack missions against the player once they are confidently established (level 5 precincts). Their bases level up slower than orks/other bases to reach that point.
• GSC Bases are harder to detect, and do not send up fighters to intercept the player.
• GSC make use of Taurox and Truck patrols and convoys.
• GSC make use of captured imperial transports and other ships.

WIP:
• Slaanesh Craft added, 3 small-medium variants for abduction and terror missions. Very decadent, opulent and depraved. Using a new tileset and mapped out courtesy of Espartano´s great work.
